inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java

```diff
@@ -147,6 +153,8 @@ public class AuditServiceImpl implements AuditService {
 
     @Value("${audit.query.source}")
     private String auditQuerySource;
+    @Value("${audit.query.url:http://127.0.0.1}")
```

Comment:
It is recommended that the default value be consistent with the configuration file and use http://127.0.0.1:10080.

Reply:
Done.